pro/src/components/CollectiveOfferLayout/CollectiveOfferLayout.module.scss

```diff
@@ -26,6 +26,7 @@
 
   &-sub-heading {
     max-width: rem.torem(588px);
+    display: block;
```

Comment:
si c'est pour mettre un display block, pourquoi ne pas le mettre dans une div directement ?

Reply:
Ce n'est pas valide, tu verras en testant ce petit bout de code au validateur https://validator.w3.org/nu/#textarea
```
<!DOCTYPE html>
    <html lang="en">
    <head>
    <meta charset="UTF-8" >
    <title>Document</title>
    </head>
    <body>
       <h1>Créer une offre <div>Mon offre</div></h1>
    </body>
</html>
```

Du coup je viens de découvrir [la dernière version de la spec](https://html.spec.whatwg.org/multipage/grouping-content.html#the-div-element) qui explique que `<div>` peut être utilisé partout où un [flow-conten](https://html.spec.whatwg.org/multipage/dom.html#flow-content-2)t est attendu